freqtrade/main.py

```diff
@@ -247,8 +256,14 @@ def app(config: dict) -> None:
                 time.sleep(1)
             elif new_state == State.RUNNING:
                 _process()
-                # We need to sleep here because otherwise we would run into bittrex rate limit
-                time.sleep(25)
+                if misc.args.backtesting:
```

Comment:
Please refactor this that backtesting is a own application state, (e.g.: `State.BACKTESTING`)